src/io/flutter/preview/PreviewView.java

```diff
@@ -84,35 +81,30 @@
   @NotNull
   private final FlutterDartAnalysisServer flutterAnalysisServer;
 
-  final QuickAssistAction actionCenter;
```

Comment:
All this code for assists is still around it just got moved into a separate toolbar class for reuse across multiple locations.
The reuse across multiple locations isn't actually used in the enabled code so I could go through and pull out that refactor from this CL but the refactor seems reasonable and should be useful in the future showing widget refactors directly in the code editor when the outline view is not enabled.